test/integration/IndividualClaims/submitClaim.js

```diff
@@ -1,36 +1,360 @@
-const { expectEvent, expectRevert, ether, time } = require('@openzeppelin/test-helpers');
-const { assert } = require('chai');
-const { ProposalCategory } = require('../utils').constants;
-const { hex } = require('../utils').helpers;
-const { submitProposal } = require('../utils').governance;
-const { buyCover, coverToCoverDetailsArray } = require('../utils').buyCover;
-const { getQuoteSignature } = require('../utils').getQuote;
-const { enrollMember, enrollClaimAssessor } = require('../utils/enroll');
-const { toBN } = web3.utils;
-
-const MCR = artifacts.require('MCR');
-const OwnedUpgradeabilityProxy = artifacts.require('OwnedUpgradeabilityProxy');
-const PooledStaking = artifacts.require('LegacyPooledStaking');
-const NXMaster = artifacts.require('NXMaster');
-
-const [owner, emergencyAdmin, unknown, member1, member2, member3, coverHolder] = accounts;
-
-const coverTemplate = {
-  amount: 1, // 1 eth
-  price: '30000000000000000', // 0.03 eth
-  priceNXM: '10000000000000000000', // 10 nxm
-  expireTime: '8000000000',
-  generationTime: '1600000000000',
-  currency: hex('ETH'),
-  period: 60,
-  contractAddress: '0xC0FfEec0ffeeC0FfEec0fFEec0FfeEc0fFEe0000',
-  asset: '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE',
-  type: 0,
+const { ethers } = require('hardhat');
+const { constants: { ZERO_ADDRESS } } = require('@openzeppelin/test-helpers');
+const { expect } = require('chai');
+
+const { daysToSeconds } = require('../../unit/IndividualClaims/helpers');
+
+const { mineNextBlock, setNextBlockTime } = require('../../utils/evm');
+const { BigNumber } = require('ethers');
+
+const { parseEther } = ethers.utils;
+
+const setTime = async timestamp => {
+  await setNextBlockTime(timestamp);
+  await mineNextBlock();
 };
 
+const priceDenominator = '10000';
+
 describe('submitClaim', function () {
-  it('submits claim and approves claim', async function () {
 
+  function calculateFirstTrancheId (lastBlock, period, gracePeriod) {
+    return Math.floor((lastBlock.timestamp + period + gracePeriod) / (91 * 24 * 3600));
+  }
+
+  it('submits ETH claim and approves claim', async function () {
+    const { DEFAULT_PRODUCT_INITIALIZATION } = this;
+    const { ic, cover, stakingPool0, as, tk } = this.withEthers.contracts;
+    const [ coverBuyer1, staker1, staker2 ] = this.accounts.members;
+
+    const productId = 0;
+    const payoutAsset = 0; // ETH
+    const period = 3600 * 24 * 30; // 30 days
+    const gracePeriod = 3600 * 24 * 30;
+
+    const amount = parseEther('1');
+
+    const assessmentStakingAmount = parseEther('1000');
+    const stakingAmount = parseEther('100');
+    await tk.connect(this.accounts.defaultSender).transfer(staker1.address, stakingAmount);
+    await tk.connect(this.accounts.defaultSender).transfer(staker2.address, stakingAmount);
+
+    const lastBlock = await ethers.provider.getBlock('latest');
+    const firstTrancheId = calculateFirstTrancheId(lastBlock, period, gracePeriod);
```

Comment:
you repeat this initialization in all tests. please move it into a function that you can call instead of repeating the code